sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
> I see. I guess @leosartaj is on a different boat with that one.
There is too much variety in coding styles (including hanging
indents) even in Sympy.

I am not an expert on PEP8 or other coding styles. I started using flake8 to solve the issue. It complained a lot about lines being under-indented or over-indented in some cases. So the changes. Well, it can be decided to not use hanging indents and go for 4-indents.